sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/query/ParallelDocumentQueryExecutionContext.java

```diff
@@ -42,7 +42,8 @@
  */
 public class ParallelDocumentQueryExecutionContext<T extends Resource>
         extends ParallelDocumentQueryExecutionContextBase<T> {
-    private CosmosQueryRequestOptions cosmosQueryRequestOptions;
+    private final CosmosQueryRequestOptions cosmosQueryRequestOptions;
+    private Map<PartitionKeyRange, String> partitionKeyRangeToContinuationTokenMap = new HashMap<>();
```

Comment:
readonly reference. Also please initialize in the constructor instead. This syntax get's confusing when you mix it with the constructor syntax.

Reply:
Done